CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added a new Traffic Ops endpoint to `GET` capacity and telemetry data for CDNi integration.
 - Added a Traffic Ops endpoints to `PUT` a requested configuration change for a full configuration or per host and an endpoint to approve or deny the request.
 - Traffic Monitor config option `distributed_polling` which enables the ability for Traffic Monitor to poll a subset of the CDN and divide into "local peer groups" and "distributed peer groups". Traffic Monitors in the same group are local peers, while Traffic Monitors in other groups are distibuted peers. Each TM group polls the same set of cachegroups and gets availability data for the other cachegroups from other TM groups. This allows each TM to be responsible for polling a subset of the CDN while still having a full view of CDN availability. In order to use this, `stat_polling` must be disabled.
+- Traffic Router: Add support for `file://`-protocol URLs for the `geolocation.polling.url` for the Geolocation database.
```

Comment:
In Java, the `//` is just interpreted as redundant path components for absolute paths. `file:` can be used with a relative paths, so the `//` should be removed.

Reply:
> In Java, the `//` is just interpreted as redundant path components for absolute paths. `file:` can be used with a relative paths, so the `//` should be removed.

I think actually the first `//` is used to specify a host ("authority"), so `///` is a synonym for `//localhost/` because the file host is implicitly understood to be the local machine when the hostname is empty. According to [RFC8089](https://datatracker.ietf.org/doc/html/rfc8089) all paths are absolute, and the local path part of a file URI must begin with `/`, so `file:///my/file/path`, `file://localhost/my/file/path` and `file:/my/file/path` are all valid and point to the same file, but `file:my/file/path` is invalid.

I can drop the `//`, though, since although that's the most common usage it's not required and `file:` is sufficient to name the protocol.

Reply:
Works for me. They aren't totally the same, since using `file://` with a relative path makes the connection an `FtpURLConnection`, but that's an edge case, so keeping `//` makes sense.